src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.jsx

```diff
@@ -1150,11 +1154,12 @@ describe('VAOS <VAFacilityPage>', () => {
       let firstRadio = screen.container.querySelector('.form-radio-buttons');
       expect(firstRadio).to.contain.text('Closest facility');
 
-      fireEvent.change(screen.getByLabelText('Sort facilities'), {
-        target: {
-          value: 'alphabetical',
-        },
+      const sel = await screen.findByTestId('facilitiesSelect');
+      // call VaSelect custom event for onChange handling
+      sel.__events.vaSelect({
+        detail: { value: 'alphabetical' },
       });
+
       await screen.findAllByRole('radio');
       expect(screen.baseElement).to.contain.text('Alphabetically');
```

Comment:
Verify it is being sorted by accessing VASelect value. The baseElement reads all the dropdown options not necessarily the selected

```suggestion
     expect(sel.value).to.equal('alphabetical');
```